docs/unified-test-documentation/dasharo-security/208-reset-to-defaults.md

```diff
@@ -0,0 +1,49 @@
+# Dasharo Security: Reset to Defaults option
```

Comment:
@Pokisiekk why we add this test to module `dasharo security`?

Reply:
I wasn't sure where to put it, but because this option applies to almost every functionality in Dasharo Security, I decided to put it in this module. OF course, I can move it to Dasharo Compatibility if you think it's more appropriate.